ansible/roles/docker/templates/ubuntu1804_sharedlibs.Dockerfile.j2

```diff
@@ -58,7 +58,7 @@ ENV OPENSSL300DIR /opt/openssl-3.0.0
 
 RUN mkdir -p /tmp/openssl_3.0.0 && \
     cd /tmp/openssl_3.0.0 && \
-    curl -sL https://www.openssl.org/source/openssl-3.0.0-alpha13.tar.gz | tar zxv --strip=1 && \
+    curl -sL https://github.com/quictls/openssl/archive/refs/tags/openssl-3.0.0-alpha13.tar.gz | tar zxv --strip=1 && \
```

Comment:
Does this contain the quic patches?

https://github.com/quictls/openssl/releases/tag/openssl-3.0.0-alpha13 points to https://github.com/quictls/openssl/commit/88df2c0b3d6162971304c06a240deb9320c9ae67 and I don't see any of the quic commits in https://github.com/quictls/openssl/commits/openssl-3.0.0-alpha13 (compared to https://github.com/quictls/openssl/commits/openssl-3.0.0-alpha13+quic).

Reply:
I thought they would be included but perhaps they are not. I'll take a closer look. And yes we should hold off a day or tow so that we can update to alpha14 :+1:

Reply:
Is it alright to use git clone instead?

Reply:
Go for it 😄